Realm/Realm.Fody/Analytics.cs

```diff
@@ -222,8 +218,8 @@ private static void ComputeHostOSNameAndVersion(out string name, out string vers
                     name = "windows";
                     break;
                 default:
-                    name = "osx";  // proved "windows" detected so default to "osx" for now
-                                   //                    name = Environment.OSVersion.Platform.ToString();
+                    name = "osx";  //// proved "windows" detected so default to "osx" for now
```

Comment:
double commenting here looks like an oversight

Reply:
This is actually on purpose to silence a StyleCop warning - it doesn't it like when single line comments span multiple lines. By doubly commenting it, you tell it that you know what you're doing, so it stops complaining 😄

Reply:
wow, that's a weird default rule! might be worth hard-disabling it.

Reply:
Yeah, I wasn't too worried about it because this code is going to be heavily reworked as part of the MongoDB Realm effort, so did the minimum amount of effort required to suppress the warnings there 😅